permits/views.py

```diff
@@ -393,6 +393,16 @@ def handle_amend_form_submission(self, form):
 
         messages.success(self.request, success_message)
 
+        if form.instance.status == models.PermitRequest.STATUS_RECEIVED:
+            data = {
+                "subject": _("Votre annonce a été prise en compte et classée"),
+                "users_to_notify": [form.instance.author.user.email],
+                "template": "permit_request_received.txt",
+                "permit_request": form.instance,
+                "absolute_uri_func": self.request.build_absolute_uri,
+            }
+            services._send_email_notification(data)
```

Comment:
_send_email_notification is marked as a protected function. Please create a public service function or rename this one as public (without "_").

Reply:
Solved!